docker/Dockerfile

```diff
@@ -100,6 +100,10 @@ RUN mkdir -p $MINIFI_BASE_DIR
 COPY --from=builder ${MINIFI_HOME} ${MINIFI_HOME}
 RUN chown -R ${USER}:${USER} /opt/minifi
 
+# Link crypto libs to known locations for civet
+RUN ln -s /usr/lib/libcrypto.so.1.0.0 /usr/lib/libcrypto.so
```

Comment:
why not set the make option with CIVET? It seems like a bandaid that could come apart if something within the container changed. Could it be such that our make CRYPTO_LIB was determined via an embedded command?